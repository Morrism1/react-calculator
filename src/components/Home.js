import Jumbotron from 'react-bootstrap/Jumbotron';

export default function Home() {
  return (
    <>
      <h1>Math Magician</h1>
      <Jumbotron>
        <h3>Welcome to our Page</h3>
        <p>
          Lorem, ipsum dolor sit amet consectetur adipisicing elit.
          Alias natus molestiae voluptatem
          similique harum nemo
          velit itaque a sequi hic laudantium, labore consectetur nihil exercitationem
          voluptates assumenda impedit error
          nesciunt!lorem
          lore nesciunt!lorem Lorem ipsum dolor sit,
          amet consectetur adipisicing elit. Commodi sapiente natus expedita,
          ullam recusandae, voluptates modi at, culpa omnis animi nobis est soluta quibusdam.
          Velit dolor esse pariatur at blanditiis?
        </p>
      </Jumbotron>
    </>
  );
}
