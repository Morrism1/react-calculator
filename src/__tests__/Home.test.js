import React from 'react';
import { render } from '@testing-library/react';

import Home from '../components/Home';
import '@testing-library/jest-dom/extend-expect';

describe('Home has ', () => {
  it('should have Math Magician when page loads', () => {
    const { getByText } = render(<Home />);
    const title = getByText('Math Magician');
    expect(title).toBeInTheDocument();
  });
  it('same content with the snapshot', () => {
    const { getByText } = render(<Home />);
    const title = getByText('Math Magician');
    expect(title).toMatchSnapshot();
  });
});
