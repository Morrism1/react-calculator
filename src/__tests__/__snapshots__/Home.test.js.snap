// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home has  same content with the snapshot 1`] = `
<h1>
  Math Magician
</h1>
`;
